frontend/src/pages/ProjectSettings/ProjectSettings.tsx

```diff
@@ -91,128 +94,133 @@ const ProjectSettings = () => {
-													>
-														{editProjectSettingsLoading ? (
-															<CircularSpinner
-																style={{
-																	fontSize: 18,
-																	color: 'var(--text-primary-inverted)',
-																}}
-															/>
-														) : (
-															'Save changes'
-														)}
-													</Button>
-												</Box>
-												<BorderBox>
-													<Stack gap="8">
-														<ErrorSettingsForm />
-														<Box borderTop="dividerWeak" />
-														<ErrorFiltersForm />
-													</Stack>
-												</BorderBox>
-												<FilterExtensionForm />
-												<SourcemapSettings />
-												<AutoresolveStaleErrorsForm />
-											</Stack>
-										),
-									},
-									{
-										key: 'general',
-										title: 'General',
-										panelContent: <DangerForm />,
-									},
-								]}
-							/>
-						</ProjectSettingsContextProvider>
-					</div>
-				</Box>
+													{editProjectSettingsLoading ? (
+														<CircularSpinner
+															style={{
+																fontSize: 18,
+																color: 'var(--text-primary-inverted)',
+															}}
+														/>
+													) : (
+														'Save changes'
+													)}
+												</Button>
+											</Box>
+											<ExcludedUsersForm />
+											<FilterSessionsWithoutErrorForm />
+											<RageClicksForm />
+											<NetworkRecordingForm />
+										</Stack>
+									),
+								},
+								{
+									key: 'errors',
+									title: 'Error monitoring',
+									panelContent: (
+										<Stack>
+											<Box
+												display="flex"
+												gap="8"
+												justifyContent="space-between"
+												alignItems="center"
+											>
+												<Text
+													size="large"
+													weight="bold"
+												>
+													Error monitoring
+												</Text>
+												<Button
+													onClick={onSubmit(
+														'error monitoring',
+													)}
+													trackingId="ProjectSettingsUpdate"
+												>
+													{editProjectSettingsLoading ? (
+														<CircularSpinner
+															style={{
+																fontSize: 18,
+																color: 'var(--text-primary-inverted)',
+															}}
+														/>
+													) : (
+														'Save changes'
+													)}
+												</Button>
+											</Box>
+											<BorderBox>
+												<Stack gap="8">
+													<ErrorSettingsForm />
+													<Box borderTop="dividerWeak" />
+													<ErrorFiltersForm />
+												</Stack>
+											</BorderBox>
+											<FilterExtensionForm />
+											<SourcemapSettings />
+											<AutoresolveStaleErrorsForm />
+										</Stack>
+									),
+								},
+								{
+									key: 'services',
+									title: 'Services',
+									panelContent: <ServicesTable />,
+									// TODO: remove when ready to enable to customers
+									hidden: !isHighlightAdmin,
```

Comment:
Confirmed this is not visible when I'm logged into my non highlight work email.

Reply:
An alternative approach here would have been to just assign tabs to an array and append this tab if `isHighlightAdmin` is true. That way we wouldn't need `hidden` prop.

Reply:
Yeah, I was thinking of that approach, but thought this may be cleaner and allow us more flexibility in the future than trying to append tabs if we have multiple cases